Modules/Microsoft365DSC/Examples/Resources/IntuneMobileThreatDefenseConnector/1-Create.ps1

```diff
@@ -0,0 +1,48 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+
+    Import-DscResource -ModuleName Microsoft365DSC
+
+    node localhost
+    {
+        IntuneMobileThreatDefenseConnector "IntuneMobileThreatDefenseConnector-Microsoft Defender for Endpoint"
+        {
+            AllowPartnerToCollectIosApplicationMetadata         = $False;
```

Comment:
Missing auth (ApplicationId, TenantId and CertificateThumbprint) parameters for the example to be able to compile.